File: starterpack/Javascript/RandomBot.js
const {
  Move,
} = require('./blitz2k18.js');
const Networking = require('./networking.js');

const network = new Networking('RandomJavaScriptBot');

network.on('map', (gameMap, id) => {
  const moves = [];

  for (let y = 0; y < gameMap.height; y++) {
    for (let x = 0; x < gameMap.width; x++) {
      const loc = { x, y };
      const { owner } = gameMap.getSite(loc);
      if (owner === id) {
        moves.push(new Move(loc, Math.floor(Math.random() * 5)));
      }
    }
  }

  network.sendMoves(moves);
});

File: starterpack/Python/RandomBot.py
import blitz2k18
from blitz2k18 import NORTH, EAST, SOUTH, WEST, STILL, Move, Square
import random


myID, game_map = blitz2k18.get_init()
hlt.send_init("RandomPythonBot")

while True:
    game_map.get_frame()
    moves = [Move(square, random.choice((NORTH, EAST, SOUTH, WEST, STILL))) for square in game_map if square.owner == myID]
    hlt.send_frame(moves)
